src/base/rss/rss_autodownloader.cpp

```diff
@@ -217,6 +217,22 @@ void AutoDownloader::removeRule(const QString &ruleName)
     }
 }
 
+QList<Article *> AutoDownloader::matchingArticles(const QString &ruleName)
```

Comment:
Sorry, it shouldn't be here. It contains only trivial logic implemented using publicly available methods. So it's some kind of "bloat" in AutoDownloader interface.